content/docs/release/_index.md

```diff
@@ -70,6 +70,9 @@ Description: >
 - [#1736 - [BUG]: failed to provision volume with StorageClass error generating accessibility requirements: no available topology found](https://github.com/dell/csm/issues/1736)
 - [#1762 - [BUG]: CSM Operator samples are incomplete](https://github.com/dell/csm/issues/1762)
 - [#1861 - [BUG]: Update the OTEL image version in operator and helm sample files](https://github.com/dell/csm/issues/1861)
+- [#1930 - [BUG]: Powerstore has unnecessary sharedNFS related codes that affects performance](https://github.com/dell/csm/issues/1930)
+- [#1942 - [FEATURE]: Release support for CSM 1.14.1](https://github.com/dell/csm/issues/1942)
```

Comment:
This doesn't seem correct. We need to add github issue for the NVMe defect and reference it here.